src/Validation/Conflict/FieldContext.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Digia\GraphQL\Validation\Conflict;
+
+use Digia\GraphQL\Language\Node\FieldNode;
+use Digia\GraphQL\Type\Definition\CompositeTypeInterface;
+use Digia\GraphQL\Type\Definition\Field;
+
+class FieldContext
+{
+    /**
+     * @var CompositeTypeInterface|null
```

Comment:
`?ComposerTypeInterface`

Reply:
I had it like this, but I got errors for it. Can't remember what complained about them.